examples/bilap.jl

```diff
@@ -0,0 +1,64 @@
+# Bi-objective linear assignment problem (bilap)
+#
+# Example 9.38 (from Ulungu and Teghem, 1994), page 255 of
+# Multicriteria Optimization (2nd edt), M. Ehrgott, Springer 2005.
+#
+# Adapted from vOptGeneric
+
+
+# ---- Packages to use
+using JuMP
+import GLPK
+import MultiObjectiveAlgorithms as MOA
+
+
+# ---- Values of the instance to solve
+C1 = [  5  1  4  7 ;  # coefficients's vector of the objective 1
+        6  2  2  6 ;
+        2  8  4  4 ;
+        3  5  7  1   ]
+
+C2 = [  3  6  4  2 ;  # coefficients's vector of the objective 2
+        1  3  8  3 ;
+        5  2  2  3 ;
+        4  2  3  5   ]
+
+n  = size(C2,1)       # number of lines/columns
+
+
+# ---- setting the model
+bilap = Model()
+@variable(bilap, x[1:n,1:n], Bin )
+@expression(bilap, objective1, sum( C1[i,j]*x[i,j] for i=1:n,j=1:n ))
+@expression(bilap, objective2, sum( C2[i,j]*x[i,j] for i=1:n,j=1:n ))
+@objective(bilap, Min, [objective1, objective2])
+@constraint(bilap , cols[i=1:n], sum(x[i,j] for j=1:n) == 1 )
+@constraint(bilap , rows[j=1:n], sum(x[i,j] for i=1:n) == 1 )
+
+
+# ---- Invoking the algorithm (Epsilon Constraint method) and the IP solver (GLPK) 
+set_optimizer(bilap, () -> MOA.Optimizer(GLPK.Optimizer))
+set_optimizer_attribute(bilap, MOA.Algorithm(), MOA.EpsilonConstraint())
+set_optimizer_attribute(bilap, MOA.ObjectiveAbsoluteTolerance(1), 1) # ugly name for this attribute
```

Comment:
Should we make this the default for `EpsilonConstraint`, so that in most cases users don't have to set this attribute?